core/jvm/src/main/scala/zio/App.scala

```diff
@@ -50,7 +50,7 @@ trait App extends DefaultRuntime {
    */
   // $COVERAGE-OFF$ Bootstrap to `Unit`
   final def main(args0: Array[String]): Unit =
-    sys.exit(
+    try sys.exit(
```

Comment:
The actual fix. Taken from https://github.com/zio/zio/issues/867#issuecomment-495580505.
Thanks :+1: